sub-logs/abderrazak-bahaj.json

```diff
@@ -0,0 +1,17 @@
+{
+  "repo": "https://github.com/abderrazak-Bahaj/Abderrazak",
+
+  "owner": {
+    "username": "abderrazak",
+    "email": "bahajabde0@gmail.com"
+  },
+
+  "target": {
+    "CNAME": {
+      "name": "portfoloi", 
+      "value": "abderrazak-Baha.github.io"
```

Comment:
I don't see anything on the GitHub page (404).

Additionally a description of the use isn't provide as well.

Reply:
@DEV-DIBSTER We don't need the website to be reachable (is-a-good-dev doesn't require it)